craft_providers/multipass/_launch.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Multipass Provider."""
+
+import logging
+
+from craft_providers import Base
+
+from .multipass_instance import MultipassInstance
+
+logger = logging.getLogger(__name__)
+
+
+def launch(
+    name: str,
+    *,
+    base_configuration: Base,
+    cpus: int = 2,
+    disk_gb: int = 256,
```

Comment:
256G was the default in Snapcraft.  I'd think 5G would definitely hit issues. 

The multipass disks are sparse so it shouldn't be significantly different on disk.  I'd rather lean towards having a bit too much to prevent people from having to figure out how to configure it.  But I agree perhaps 256G is a bit too much if the user uses it often and never cleans.

@sergiusens Do you by chance have any data on a "large project" built by Snapcraft?

Reply:
Launchpad has a 60G quota.  So I'll change this 64G for now :D